src/core/documentation/decision.ts

```diff
@@ -0,0 +1,63 @@
+import { Element } from "../model/element";
+import { Format } from "./format";
+import { DecisionStatus } from "./decisionStatus";
+
+export interface DecisionDto {
```

Comment:
I always also struggled with myself if I should add interfaces for the DTOs, in order to get some better support (e.g. auto completion) when writing toDto and fromDto methods. I finally settled with not adding the interfaces, since they will show up on the API exposed to the end user, and I dont want that.

What you could do, is to define, but not export the interface, so you'll at least have typing support within this single file

Reply:
I think i'll be consistent with the rest of the library, I feel typing isnt really too nessesary here.